# std_msgs/Header header
string controllerType
---
bool finishSwitchController